compose/src/main/java/com/bottlerocketstudios/compose/snippets/SnippetUiModel.kt

```diff
@@ -3,6 +3,8 @@ package com.bottlerocketstudios.compose.snippets
 import com.bottlerocketstudios.compose.util.StringIdHelper
 
 data class SnippetUiModel(
+    val id: String,
+    val workspaceSlug: String,
     val title: String,
```

Comment:
Added these in preparation for Snippet Detail view, but it got complictaed enough that I felt it should be a different PR.